src/gui/presets.c

```diff
@@ -55,7 +55,7 @@ static const char *_dt_gui_presets_aperture_value_str[]
 
 // format string and corresponding flag stored into the database
 static const char *_dt_gui_presets_format_value_str[5]
-    = { N_("normal images"), N_("raw"), N_("HDR"), N_("monochrome"), N_("color") };
+    = { N_("non-raw images"), N_("raw files"), N_("HDR"), N_("monochrome"), N_("color") };
```

Comment:
why not "raw images" to be consistent with "non-raw images" ?

Reply:
Raw files are not images.

Reply:
Indeed !

Reply:
So maybe just:  "images" and "raw files" ?

Reply:
this is deliberate double impact on my side.

all images are non-raw. that's why "normal images" was perfectly valid name... but people got confused so "non-raw images"
raws are essentially sensor data-dumps + metadata. And without processing the sensor dump is hardly viewable as image so it's a file then.
It'd be great if we worked towards consistency to help user mental load with that: raws aren't images and images aren't raws and you can get image from raw but can't get raw from image. So ultimately in some future ver the wording could be simplified to "images" & "raws" but we're not at that point yet.

Reply:
@elstoc gave me good idea. lemme test it with you.

Reply:
removed redundancy. now we have it super obvious: "non-raw" and "raw". That should clear ALL confusion :) and let's hope I'm wrong about double impact being needed :)